src/components/About/index.js

```diff
@@ -1,273 +1,398 @@
+                district/state
+              </li>
               <li>Ticketed events in the district/state</li>
               <li>Staff office hours</li>
-              <li>Opportunities in Washington, D.C. to speak with your representative</li>
+              <li>
+                Opportunities in Washington, D.C. to speak with your
+                representative
+              </li>
             </ul>
             <p>We do not currently include:</p>
             <ul>
               <li>Fundraisers outside the district/state</li>
-              <li>Inappropriate events to ask policy questions (e.g. funerals)</li>
+              <li>
+                Inappropriate events to ask policy questions (e.g. funerals)
+              </li>
               <li>Campaign events where the member will not take questions</li>
             </ul>
           </div>
           <div className="col-sm-6">
             <h3>What to expect</h3>
-            <p>The most powerful thing you can do, as a constituent, is ask an earnest, pressing question on an issue
-              close
-              to you.
-                <strong>Your personal story is incredibly valuable.</strong> It’s precisely how sometimes dry policy is
-                              connected with
-                              the lives of real people. It’s not always easy to speak up, but these times call for courage in all of us.
-                Take the mic and tell your representative why she or he needs to act in your best interest.</p>
-            <p>The Town Hall Project strongly encourages you to only attend and ask questions of your own
-              representatives. Remember
-                that any town hall has limited time, and a question you take is one less question for a constituent.</p>
+            <p>
+              The most powerful thing you can do, as a constituent, is ask an
+              earnest, pressing question on an issue close to you.
+              <strong>Your personal story is incredibly valuable.</strong> It’s
+              precisely how sometimes dry policy is connected with the lives of
+              real people. It’s not always easy to speak up, but these times
+              call for courage in all of us. Take the mic and tell your
+              representative why she or he needs to act in your best interest.
+            </p>
+            <p>
+              The Town Hall Project strongly encourages you to only attend and
+              ask questions of your own representatives. Remember that any town
+              hall has limited time, and a question you take is one less
+              question for a constituent.
+            </p>
 
             <p>For more, we recommend:</p>
             <ul>
-              <li>Indivisible's
-                  <a href="https://indivisible.org/resource/town-hall-guide" target="_blank">Town Hall strategy resources</a>
+              <li>
+                Indivisible's
+                <a
```

Comment:
can you make sure there is a space between 'Indivisible's' and 'Town Hall strategy resources'? Currently on the site they're smooshed together